internal/bminventory/inventory.go

```diff
@@ -329,6 +329,32 @@ func (b *bareMetalInventory) formatIgnitionFile(cluster *common.Cluster, params
 	return res, nil
 }
 
+func (b *bareMetalInventory) formatNextStepRunnerCommand(clusterID string, hostID string) (string, *[]string) {
+
+	arguments := []string{"run", "--rm", "-ti", "--privileged", "--pid=host", "--net=host",
```

Comment:
There is no way to run podman in podman.  How should this work?  Seems problematic

Reply:
The agent side uses `nsenter` to run commands, like our other tools. See https://github.com/openshift/assisted-installer-agent/pull/47. These two PRs has been tested end-to-end all the way including installation, reboots, etc.